lib/src/mui_suggestors/components/mui_inline_alert_migrator.dart

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react_codemod/src/mui_suggestors/constants.dart';
+import 'package:over_react_codemod/src/util/component_usage.dart';
+import 'package:over_react_codemod/src/util/component_usage_migrator.dart';
+
+import 'mui_migrator.dart';
+
+class MuiInlineAlertMigrator extends ComponentUsageMigrator
+    with MuiMigrator, AlertDisplayPropsMigrator {
+  static const toastAlertProps = [
+    'dismissAfter',
+    'initiallyShown',
+    'isDismissible',
+    'onDidHide',
+    'onDidShow',
+    'onWillHide',
+    'onWillShow',
+    'transition',
+    'transitionCount',
+    'transitionInCount',
+    'transitionOutCount',
+  ];
+
+  @override
+  bool shouldMigrateUsage(FluentComponentUsage usage) =>
+      usesWsdFactory(usage, 'Alert') &&
+      // If the alert has props related to the `toast` variant, don't try to
+      // migrate it.
+      !usage.cascadedProps
+          .any((prop) => toastAlertProps.contains(prop.name.name));
+
+  @override
+  void migrateUsage(FluentComponentUsage usage) {
+    super.migrateUsage(usage);
+
+    yieldPatchOverNode('$muiNs.Alert', usage.factory!);
+
+    handleCascadedPropsByName(usage, {
+      'heading': (p) => migrateAlertHeading(usage, p),
+      'headingProps': yieldPropManualMigratePatch,
+      'skin': migrateAlertSkin,
+      'size': migrateAlertSize,
+    });
+  }
+}
+
+mixin AlertDisplayPropsMigrator on ComponentUsageMigrator {
+  void migrateAlertSkin(PropAssignment prop) {
+    final rhs = prop.rightHandSide;
+
+    final severityFromWsdSkin = mapWsdConstant(rhs, const {
+      'AlertSkin.DEFAULT': '$muiNs.AlertSeverity.info',
+      'AlertSkin.SUCCESS': '$muiNs.AlertSeverity.success',
+      'AlertSkin.WARNING': '$muiNs.AlertSeverity.warning',
+      'AlertSkin.DANGER': '$muiNs.AlertSeverity.error',
+    });
+
+    if (severityFromWsdSkin != null) {
+      yieldPropPatch(
+        prop,
+        newName: 'severity',
+        newRhs: severityFromWsdSkin,
+      );
+      return;
+    }
+
+    final usesInverseSkin =
+        isWsdStaticConstant(prop.rightHandSide, 'AlertSkin.INVERSE');
+    var migrateVariantToGray = usesInverseSkin ||
+        isWsdStaticConstant(prop.rightHandSide, 'AlertSkin.GRAY');
+
+    mapWsdConstant(rhs, const {
+      'AlertSkin.GRAY': '$muiNs.AlertVariant.wsdGray',
+    });
```

Comment:
This call has no side-effects, and the return value is unused, so this should essentially be a no-op. Can we remove it?

Reply:
We can; missed removing that during a clean up refactor. Thank you for catching it!